src/repo/mod.rs

```diff
@@ -116,6 +117,13 @@ pub trait DataStore: PinStore + Debug + Send + Sync + Unpin + 'static {
     async fn wipe(&self);
 }
 
+#[async_trait]
```

Comment:
This seems unnecessary, or is there a reason why this needs to be an `#[async_trait]` as well which is not obvious?

Reply:
Yep, it's not needed 👍